### PR TITLE
updates recipies before new production

### PR DESCRIPTION
**Pull request by camendola**
* METFilters: the new recommendation is to rerun the flag ecalBadCalibFilter
* JECUncertainty updated
* TES updated

### CODE REVIEW

File: NtupleProducer/plugins/HTauTauNtuplizer.cc [hunk 1]
```diff
@@ -2195,23 +2195,49 @@ void HTauTauNtuplizer::analyze(const edm::Event& event, const edm::EventSetup& e
   eSetup.get<JetCorrectionsRecord>().get("AK4PFchs",JetCorParColl);
   JetCorrectorParameters const & JetCorPar = (*JetCorParColl)["Uncertainty"];
   JetCorrectionUncertainty jecUnc (JetCorPar);
+  bool RunB = (_runNumber > 297046 && _runNumber < 299329);
+  bool RunC = (_runNumber > 299368 && _runNumber < 302029);
+  bool RunD = (_runNumber > 302030 && _runNumber < 303434);
+  bool RunE = (_runNumber > 303824 && _runNumber < 304797);
+  bool RunF = (_runNumber > 305040 && _runNumber < 306462);
```
Comment (francescobrivio) [inline NtupleProducer/plugins/HTauTauNtuplizer.cc]: @camendola shouldn't we have some >= or <= here?

Comment (francescobrivio) [reply NtupleProducer/plugins/HTauTauNtuplizer.cc]: checking on PdmV I think the all should be >= or <=
https://twiki.cern.ch/twiki/bin/view/CMS/PdmV2017Analysis

Comment (camendola) [reply NtupleProducer/plugins/HTauTauNtuplizer.cc]: ok thanks

File: NtupleProducer/python/HiggsTauTauProducer_94X.py [hunk 2]
```diff
@@ -145,6 +145,31 @@
...
+    [872439604,872422825,872420274,872423218,
+     872423215,872416066,872435036,872439336,
+     872420273,872436907,872420147,872439731,
+     872436657,872420397,872439732,872439339,
+     872439603,872422436,872439861,872437051,
+     872437052,872420649,872422436,872421950,
+     872437185,872422564,872421566,872421695,
+     872421955,872421567,872437184,872421951,
+     872421694,872437056,872437057,872437313])
+
```
Comment (francescobrivio) [inline NtupleProducer/python/HiggsTauTauProducer_94X.py:161]: @camendola I would use this list which is the recommended one

Comment (camendola) [reply NtupleProducer/python/HiggsTauTauProducer_94X.py:161]: ok

File: NtupleProducer/plugins/HTauTauNtuplizer.cc (hunk 1 above)
Comment (camendola) [reply NtupleProducer/plugins/HTauTauNtuplizer.cc]: done

**Pull request merged**